.circleci/config.yml

```diff
@@ -12,8 +12,8 @@ commands:
   # Our minimum supported rust version is specified here.
   setup-rust-min-version:
     steps:
-      # https://searchfox.org/mozilla-central/source/python/mozboot/mozboot/util.py#26
-      - run: rustup override set 1.70.0
+      # https://searchfox.org/mozilla-central/source/python/mozboot/mozboot/util.py#14
```

Comment:
@brennie Do you mean [this permalink](https://hg.mozilla.org/integration/autoland/rev/a545e84b3674) as opposed to what's supplied? Just curious. 😊

Reply:
The "problem" with a permalink is that it will always reflect a stale version. The problem with a "live" link is that the link itself might go stale, but while it's not stale it will reflect current reality.

To put it another way: should it be a link to what's *currently* on m-c, or a link that simply verifies what was there at some point in the past? The former seems mildly more useful to me, but I don't have a strong opinion here.

Reply:
If you press `y` on searchfox, it will change the URL to include the revision you're looking at. That's what I meant. eg `https://searchfox.org/mozilla-central/rev/a7e19af54ce40c7f2d0cb0457d3b12c279fe0f68/python/mozboot/mozboot/util.py#14`

Reply:
> The "problem" with a permalink is that it will always reflect a stale version. The problem with a "live" link is that the link itself might go stale, but while it's not stale it will reflect current reality.

Yep, I ran into exactly that! 😅 I almost wonder if changing the comment to say "Please make sure this version is <= `MINIMUM_RUST_VERSION` in `mozilla-central`" would be a bit better, in case m-c changes where `MINIMUM_RUST_VERSION` lives.

Reply:
Or a searchfox link for that symbol - or just a link to https://github.com/mozilla/application-services/blob/main/docs/design/rust-versions.md (which does the searchfox thang...)

Reply:
Sounds perfect, thanks all! It looks like we already link to `rust-versions.md` just above this section, right after the `commands:` heading, so I dropped this link altogether.